epersonregistrations.md

```diff
@@ -44,13 +44,16 @@ To create a new EPerson registration, perform a post with the JSON below to the
 }
 ```
 
+Requires query parameter 'type' set with either 'register' or 'forgot' value, depending on the action requested.
+
 No other properties can be set (e.g. the name cannot be defined)
 If successful, an email will be sent with a token allowing the user to continue the registration
 
 Verifying whether a new registration can be created can happen using the "epersonRegistration" [feature](features.md), verified against the site
 
 Status codes:
 * 201 Created - if the operation succeed
+* 400 Bad Request - if e.g. the e-mail address is in a domain that is not allowed in config `authentication-password.domain.valid`
```

Comment:
this should be used if the type parameter is missing or invalid (other than register / forgot) or we should state (and implement) a default to register to keep backward compatibility

Reply:
E-mail address given now not in domain will given 422 unprocessable entity error